packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayoutWithAddCardPanel/DraggingCard/DraggingCard.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+
+export interface IAddCardStyles {
+  /**
+   * Style set for the add card item
+   */
+  container: IStyle;
+  title: IStyle;
+  titleText: IStyle;
+  body: IStyle;
+}
+
+export const getStyles = (): IAddCardStyles => {
+  return {
+    container: {
+      zIndex: 12000000,
```

Comment:
>zIndex: 12000000, [](start = 6, length = 17)

can we not have this huge z index. Let us start small

Reply:
Changed to 10.